TOC.md

```diff
@@ -143,6 +143,7 @@
         - [`SET`](/data-type-string.md#set-type)
       - [JSON Type](/data-type-json.md)
     + Functions and Operators
+<<<<<<< HEAD
```

Comment:
Please resolve this conflict by referring to https://github.com/pingcap/docs-cn/pull/3863/files#diff-12810a4235487caeff6e7338176dc036R283. @zz-jason